docs/tutorials/up12_installation.rst

```diff
@@ -0,0 +1,45 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _up12_installation_tutorial:
+
+.. |propeller_name| replace:: UP12
+.. |motor_name| replace:: Vertiq 2306
+
+*********************************************
+Installing |propeller_name| on a |motor_name|
+*********************************************
+.. figure:: ../_static/tutorial_images/up12_installation/annotated_propeller.png
+    :align: center
+    :width: 750
+    :alt: Annotated |propeller_name|
+
+    Annotated |propeller_name|
+
+To install the |propeller_name| on a |motor_name| module you must first remove the main shoulder bolt from the adapter. Be sure not to lose the two washers on either side of the propeller hub. 
+
+.. figure:: ../_static/tutorial_images/up12_installation/hub_removed.jpg
+    :align: center
+    :width: 500
+    :alt: Hub removed from adapter
+
+    Adapter and Hub Separated
+
+Use the M2x6 Screws to mount the adapter to the |motor_name|.
+
+.. figure:: ../_static/tutorial_images/up12_installation/adp_mounted.jpg
+    :align: center
+    :width: 500
+    :alt: Adapter mounted to motor
+
+    Adapter mounted to motor
+
+Re-attach the propeller hub to the adapter. When tightening the main shoulder bolt, continue tightening until the hub does not spin freely. Loosen the bolt ¼ turn at a time until it again spins freely. There should be very little axial play of the hub along the main shoulder bolt.
+
+.. figure:: ../_static/tutorial_images/up12_installation/propeller.jpg
```

Comment:
do we have a picture of this without the ENG labels? 

Also, a video of what a properly seated prop looks like could be helpful. I know it's supposed to have a little wobble, but showing people how much is right could be helpful.

Reply:
I like the video ideas. Backlog task? All real pics will be replaced with nice ones from Jon